.github/workflows/ci.yml

```diff
@@ -3,6 +3,7 @@ on:
     branches:
       - main
   pull_request:
+  workflow_dispatch:
```

Comment:
why was this added?

Reply:
To make it possible to run tests in branches in forks.